app/server/src/routes/apps.ts

```diff
@@ -14,5 +14,6 @@ router.post(
     bodyParser.text(),
     SessionsController.postSessionLabel
 );
+router.get("/:appName/sessions/:id", SessionsController.getSession);
```

Comment:
Yeah, this is the same route I've used for the html response when you want to load an app with a given session id. I'll change my route to take a query string parameter instead. Alternatively I suppose we could distinguish api requests with an `/api`/ part.